components/Hero/StoryHeroMvp.tsx

```diff
@@ -140,10 +141,24 @@ export const StoryHeroMvp = ({
         addDarkOverlay={addDarkOverlay}
         imageOnLeft={isLeftImage}
         tabColor={paletteAccentColors[tabColorValue]}
+        caption={caption}
       />
       {!!getNumBloks(heroTexturedBar) && (
         <CreateBloks blokSection={heroTexturedBar} />
       )}
+      {caption && (
+        <Container bgColor="white" className={styles.captionWrapper}>
+          <Text
+            // id is for aria-describedby in the images since we can't use figcaption here
+            id="story-hero-caption"
+            variant="caption"
+            leading="display"
+            className={styles.caption}
+          >
```

Comment:
Add an id to the caption so it can be referenced by `aria-describedby` in the hero foreground and background images. With how the hero is structured, we can't use the figure/figcaption HTML tags.

Reply:
Will there only ever be one of these on a page? If there might be more than one, you will want a dynamic id.

Reply:
@sherakama Yup, there will always only be one single caption for the hero, so not too concerned about the uniqueness here 😄

Reply:
Will there only ever be one hero on the page?

Reply:
Yup, only one hero ever 😄 It's not a nestable blok that can be dropped in anywhere.

Reply:
Ok. Great.